src/profile/helpers/__tests__/editData.test.ts

```diff
@@ -172,16 +202,16 @@ describe('editData.ts ', () => {
         updateItemAndCreateSaveData,
```

Comment:
Off-topic: comment for line 193: my IDE suggest `!!node.primary` could be simplified to `node.primary`

Reply:
changed. Required a new commit, because this wasn't added in this PR